### PR TITLE
Out 213 SWR and minor fixes

### PR DESCRIPTION
**Pull request by aatbip**
Ref: https://linear.app/copilotplatforms/issue/OUT-170/switching-out-of-preview-mode-scrolls-the-user-down-the-page
Ref: https://linear.app/copilotplatforms/issue/OUT-212/when-i-install-client-home-clientgivenname-should-be-set-as-a-chip
Ref: https://linear.app/copilotplatforms/issue/OUT-208/add-pseudo-realtime-functionality-to-client-side-experience-via

- Adds useSWR hook for pseudo-realtime functionality
- Fixes scroll issue between mode switch
- Adds chip for `{{client.givenName}}` in the default state

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -7,28 +7,24 @@
...
   "parserOptions": {
     "ecmaVersion": "latest",
     "sourceType": "module"
   },
   "rules": {
-    "indent": [
-      "error",
-      2
-    ],
+    "indent": "off",
```
Comment (saugatmaharjan) [inline .eslintrc.json:19]: why turn off indent rule @aatbip ?

File: .prettierrc.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
+  "trailingComma": "es5",
```
Comment (saugatmaharjan) [inline .prettierrc.json:2]: is this config recommended by team? changes on this config adding unnecessary changes in other files.

**Comment by saugatmaharjan** [comment]
@aatbip address the type warning

File: .prettierrc.json (hunk 2 above)
Comment (aatbip) [reply .prettierrc.json:2]: reverted this back to `"trailingComma": "all"` @saugatmaharjan

File: .eslintrc.json (hunk 1 above)
Comment (aatbip) [reply .eslintrc.json]: `"indent" : "warn"` has been set in the next commit @saugatmaharjan

**Review by saugatmaharjan — APPROVED** [review]
